examples/access-control/README.md

```diff
@@ -0,0 +1,28 @@
+## Access Control
+This example demonstrates how Authentication and Authorization works. The example contains Jupyter with Spark as query engines, OpenFGA as Authorization backend, Keycloak as IdP and Minio as storage.
+
+Run the example with the following command:
+```bash
+cd examples/access-control
+docker compose up
+```
+
+Now open in your Browser:
+* Jupyter: [http://localhost:8888](http://localhost:8888)
+* Keycloak UI: [http://localhost:30080](http://localhost:30080)
+* Swagger UI: [http://localhost:8181/swagger-ui/#/](http://localhost:8181/swagger-ui/#/) (Note that more endpoints are available than in the Minimal example as permissions are enabled)
+* Lakekeeper UI (DON'T USE IT FOR BOOTSTRAPPING, There is a Notebook for it.): [http://localhost:8181](http://localhost:8181)
+
+Start by following the instructions in the `01-Bootstrap.ipynb` Notebook in Jupyter. After that, you can login to the [UI](http://localhost:8181) as:
+* Username: `peter`
+* Password: `iceberg`
```

Comment:
Weren't the passwords with `Iceberg` with capital i?

Reply:
No, all lowercase now. It used to be uppercase here and lowercase everywhere else. That was an error that we fixed already